docs/service-introspection-onboarding.md

```diff
@@ -149,32 +149,59 @@ by service introspection.
+pipeline in the Azure DevOps portal:
 
 ```yaml
-latest_commit=$(git rev-parse --short HEAD)
-VERSION_TO_DOWNLOAD=$(curl -s "https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/') 
-echo "Downloading SPK version $VERSION_TO_DOWNLOAD" && wget "https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
-chmod +x ./spk-linux
-./spk-linux deployment create  -n $(ACCOUNT_NAME) -k $(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Release.ReleaseId) --hld-commit-id $latest_commit --env $(Release.EnvironmentName) --image-tag $(Build.BuildId)
+latest_commit=$(git rev-parse --short HEAD) VERSION_TO_DOWNLOAD=$(curl -s
+"https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep
+"tag_name" | sed -E 's/.*"([^"]+)".*/\1/') echo "Downloading SPK version
+$VERSION_TO_DOWNLOAD" && wget
+"https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
+chmod +x ./spk-linux ./spk-linux deployment create  -n $(ACCOUNT_NAME) -k
+$(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Release.ReleaseId)
+--hld-commit-id $latest_commit --env $(Release.EnvironmentName) --image-tag
+$(Build.BuildId)
 ```
 
 This task is similar to the one from step 1 but instead passes the information
 that corresponds to the CD release pipeline.
 
-For multi-stage pipelines, paste the following yaml task towards the end of your image tag release stage:
+****Option 2:** As a **stage**** (if your setup is a
+[Multi-stage Azure Pipeline](https://devblogs.microsoft.com/devops/whats-new-with-azure-pipelines/))
+
+**Instructions:**
+
+Paste the following yaml task towards the end of your image tag release stage in
+your multi-stage `azure-pipelines.yml`:
 
 ```yaml
-latest_commit=$(git rev-parse --short HEAD)
-tag_name=$(Build.BuildId)
-VERSION_TO_DOWNLOAD=$(curl -s "https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/') 
-echo "Downloading SPK version $VERSION_TO_DOWNLOAD" && wget "https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
-chmod +x ./spk-linux
-./spk-linux deployment create  -n $(ACCOUNT_NAME) -k $(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Build.BuildId) --hld-commit-id $latest_commit --env $(Build.SourceBranchName) --image-tag $tag_name
+latest_commit=$(git rev-parse --short HEAD) tag_name=$(Build.BuildId)
+VERSION_TO_DOWNLOAD=$(curl -s
+"https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep
+"tag_name" | sed -E 's/.*"([^"]+)".*/\1/') echo "Downloading SPK version
+$VERSION_TO_DOWNLOAD" && wget
+"https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
+chmod +x ./spk-linux ./spk-linux deployment create  -n $(ACCOUNT_NAME) -k
+$(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY)  --p2 $(Build.BuildId)
+--hld-commit-id $latest_commit --env $(Build.SourceBranchName) --image-tag
+$tag_name
```

Comment:
Same here, new lines are going to cause this bash task to fail if the user just copy pastes it